lua/fzy.lua

```diff
@@ -293,9 +301,15 @@ function M.pick_one(items, prompt, label_fn, cb)
   )
   vim.cmd('startinsert!')
   local f = io.open(inputs, 'a')
-  for i, item in ipairs(items) do
-    local label = string.format(digit_fmt .. '│ %s', i, label_fn(item))
-    f:write(label .. '\n')
+  if M.opts.number_items ~= false then
+    local num_digits = math.floor(math.log(math.abs(#items), 10) + 1)
+    local digit_fmt = '%0' .. tostring(num_digits) .. 'd'
+    for i, item in ipairs(items) do
+      local label = string.format(digit_fmt .. '│ %s', i, label_fn(item))
+      f:write(label .. '\n')
+    end
+  else
+    f:write(table.concat(vim.tbl_map(label_fn, items), "\n"))
```

Comment:
Doesn't this break the on_selection callback given that the marker will be missing?

Reply:
Oh yes, thanks, I forgot about that. I will update that as well.

Reply:
I addressed this in https://github.com/mfussenegger/nvim-fzy/pull/21/commits/eb09bc119563fef45fc1f2a75d843ab59596f24d by adding a reverse lookup table from `label` to `item`, since we no longer have access to the item's index when numbers are disabled.